internal/artifacts/module_arch_test.go

```diff
@@ -302,7 +307,7 @@ builders:
 				"cf", os.Getwd)).Should(HaveOccurred())
 		})
 
-		It("Fails on platform validation", func() {
+		It("Fails on platform validation when mtad.yaml should be generated", func() {
```

Comment:
This change was meant for line 143 I think

Reply:
right, fixed